sdk/identity/azure-identity/pom.xml

```diff
@@ -19,11 +19,34 @@
     <relativePath>../../../pom.client.xml</relativePath>
   </parent>
 
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.revapi</groupId>
+        <artifactId>revapi-maven-plugin</artifactId>
+        <version>0.11.2</version> <!-- {x-version-update;org.revapi:revapi-maven-plugin;external_dependency} -->
+        <configuration>
+        <analysisConfiguration>
+          <revapi.ignore combine.children="append">
+            <item>
+              <code>java.method.added</code>
+              <class>com.azure.core.http.ProxyOptions</class>
+              <justification>The new API added in azure core can be ignored, as it doesn't break the API. </justification>
+            </item>
+          </revapi.ignore>
+        </analysisConfiguration>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
```

Comment:
Could this configuration be added into the revapi configuration JSON file?

Reply:
we'll need to make a separate file for that, which @JonathanGiles isn't covinvced about yet.
Putting it in the same JSON file will apply it to parent level config, and that will get applied to azure-core itself, i.e we will stop checking azure-core changes itself on azure-core.

We'll be removing this config from pom file, after release for now.